config/locales/en.yml

```diff
@@ -907,122 +907,122 @@ en:
-          assembly_point: "Assembly Point"
+          assembly_point: "Emergency Assembly Point"
           defibrillator: "Defibrillator"
           fire_extinguisher: "Fire Extinguisher"
           fire_water_pond: "Fire Water Pond"
           landing_site: "Emergency Landing Site"
           life_ring: "Emergency Life Ring"
           phone: "Emergency Phone"
-          siren: "Emergency Siren"
-          suction_point: "Emergency Suction Point"
-          water_tank: "Emergency Water Tank"
+          siren: "Siren"
+          suction_point: "Water Drafting Site"
+          water_tank: "Emergency Water Reservoir"
         highway:
           abandoned: "Abandoned Highway"
-          bridleway: "Bridleway"
-          bus_guideway: "Guided Bus Lane"
+          bridleway: "Bridle Path"
+          bus_guideway: "Guided Busway"
           bus_stop: "Bus Stop"
-          construction: "Highway under Construction"
-          corridor: "Corridor"
+          construction: "Highway Under Construction"
+          corridor: "Indoor Corridor"
           crossing: "Crossing"
-          cycleway: "Cycle Path"
+          cycleway: "Bicycle Path"
           elevator: "Elevator"
           emergency_access_point: "Emergency Access Point"
-          emergency_bay: "Emergency Bay"
+          emergency_bay: "Emergency Stopping Place"
           footway: "Footpath"
           ford: "Ford"
-          give_way: "Give Way Sign"
+          give_way: "Yield Sign"
           living_street: "Living Street"
           milestone: "Milestone"
-          motorway: "Motorway"
-          motorway_junction: "Motorway Junction"
-          motorway_link: "Motorway Road"
+          motorway: "Controlled-Access Highway"
+          motorway_junction: "Highway Exit"
+          motorway_link: "Highway Ramp"
           passing_place: "Passing Place"
           path: "Path"
-          pedestrian: "Pedestrian Way"
-          platform: "Platform"
+          pedestrian: "Pedestrian Mall"
+          platform: "Transit Platform"
           primary: "Primary Road"
-          primary_link: "Primary Road"
+          primary_link: "Primary Link Road"
           proposed: "Proposed Road"
-          raceway: "Raceway"
+          raceway: "Motor Racetrack"
           residential: "Residential Road"
           rest_area: "Rest Area"
           road: "Road"
           secondary: "Secondary Road"
-          secondary_link: "Secondary Road"
+          secondary_link: "Secondary Link Road"
           service: "Service Road"
-          services: "Motorway Services"
+          services: "Service Area"
           speed_camera: "Speed Camera"
-          steps: "Steps"
+          steps: "Stairway"
           stop: "Stop Sign"
           street_lamp: "Street Lamp"
           tertiary: "Tertiary Road"
-          tertiary_link: "Tertiary Road"
-          track: "Track"
+          tertiary_link: "Tertiary Link Road"
+          track: "Land Management Road or Vehicular Trail"
```

Comment:
A “track”, unqualified, is something a deer forges in the woods, a trail of shoeprints in the mud, or [maybe](https://ell.stackexchange.com/a/343841) the imprint of a farm tractor cutting through the fields. Qualified, it could be a `railway=track` (train track), `leisure=track` (running track), or `highway=raceway` (racetrack), but definitely nothing to do with `highway=track`.

`highway=track` is the only tag for which I had to equivocate on the description. This tag is essentially used for two different purposes:

1. Special-purpose roads built for a productive land-based activity (forestry roads, mining roads, levee roads, fire roads, dirt roads on a farm)
2. In some regions, quasi-roads used by offroading vehicles for recreation

There isn’t a concise, obvious word for the first purpose, let alone both at the same time. id-tagging-schema uses “Track / Land-Access Road” as a grand compromise, but this is a word salad that only makes sense to someone who managed to read all of openstreetmap/id-tagging-schema#288 and the preceding mailing list threads.

I considered a number of [terms of art](https://en.wiktionary.org/wiki/term_of_art#English) for this tag. The two that stood out to me are “land management road” and “resource road”. These are established albeit obscure terms for use case (1) in the U.S. and Canada, respectively.

An American user probably won’t have encountered the phrase “land management road” per se, but they’ll hopefully recall the Bureau of Land Management and the various activities allowed on BLM lands. But they might misunderstand “management” to imply that the road is actively maintained. Appending “vehicular trail” would counter that assumption.

The current description is very long. I’m inclined to switch to the Canadian “resource road”, which is shorter. It’s also more enigmatic, so the user is less likely to assume anything in particular about the road.